pkgs/top-level/all-packages.nix

```
@@ -14660,6 +14660,11 @@ with pkgs;
 
   redo-sh = callPackage ../development/tools/build-managers/redo-sh { };
 
+  regclient = callPackage ../development/tools/regclient { };
+  regbot = regclient.regbot;
+  regctl = regclient.regctl;
+  regsync = regclient.regsync;
```

Not sure if we need top-level attributes for all of them.

---

`regctl` is the one that I find important. Because people will search for it. Will they be listed on search if removed here?

---

no but the following is shorter and less repetetive.

```suggestion
  inherit (regclient) regbot regctl regsync;
```

---

I agree it's shorter. But `regctl` should show on searches... Any idea?

---

Done